src/js/debug.js

```diff
@@ -0,0 +1,14 @@
+var NODE_DEBUG = process.env.NODE_DEBUG;
+
+module.exports = function(tag) {
+  function debug(str) {
+    var prefix = '\033';
+    console.info(`${tag} ${prefix}[90m${str}${prefix}[0m`);
```

Comment:
What about highlight tag, and leave message unmodified?

e.g. `${prefix}[90m${tag}[0m ${str}`

Reply:
Grey messages just don't make confusion on other normal logs.